pkgs/misc/cups/drivers/splix/default.nix

```diff
@@ -1,24 +1,35 @@
-{stdenv, fetchurl, cups, zlib}:
-
-stdenv.mkDerivation rec {
-  name = "splix-2.0.0";
-
-  src = fetchurl {
-    url = "mirror://sourceforge/splix/${name}.tar.bz2";
-    sha256 = "0bwivrwwvh6hzvnycpzqs7a0capgycahc4s3v9ihx552fgy07xwp";
+{ stdenv, fetchsvn, fetchurl, cups, zlib }:
+let rev = "r315"; in
+stdenv.mkDerivation (rec {
+  name = "splix-svn-${rev}";
```

Comment:
@zimbatm that changes the name of the pkg from `splix-svn` to `splix-svn-r`. That's *bad*.